ouroboros-consensus/test-storage/Test/Ouroboros/Storage/FS/StateMachine.hs

```diff
@@ -1164,6 +1197,8 @@ tag = C.classify [
           Right $ tagPutSeekGet put (Set.insert h seek)
         Get (Handle h _) n | n > 0 && Set.member h seek ->
           Left TagPutSeekGet
+        GetAt (Handle h _) n _ | n > 0 && Set.member h seek ->
+          Left TagPutSeekGet
```

Comment:
I don't think this tag applies.

Reply:
Because the tag name has Get instead of GetAt?

I think it tags tests where Seek interacts with Get. GetAt has different properties, but we still want such tests. So maybe I should just add a different tag, like TagPutSeekGetAt?

Reply:
Ok, I thought the tag was more specific: put something, seek such that the next get reads the thing that was put, then do that get. This wouldn't work the same way for getAt. But I was mistaken.